queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/legal_entity_info.py

```diff
@@ -100,6 +100,13 @@ def update_naics_info(legal_entity: LegalEntity, naics: Dict):
 
 def get_next_corp_num(entity_type: str):
     """Retrieve the next available sequential corp-num from Lear or fallback to COLIN."""
+    # Handle firms
+    if entity_type in (
+        BusinessCommon.EntityTypes.SOLE_PROP,
+        BusinessCommon.EntityTypes.PARTNERSHIP,
+    ):
+        entity_type = "FM"
```

Comment:
Have to manually set entity_type here for SPs and GPs since [`LegalEntityType` only contains "FM"](https://github.com/bcgov/lear/blob/feature-legal-name/legal-api/src/legal_api/models/legal_entity.py#L772).

Reply:
I don't think we need to manually set it for firms(SPs & GPs) because as far as I know, incorporation filing is not for firms but for other types of business. Instead, firms do registration. And it's already passed `FM` into it for registration [here.](https://github.com/bcgov/lear/blob/feature-legal-name/queue_services/entity-filer/src/entity_filer/filing_processors/registration.py#L112) 

But need to confirm it with other devs...

Reply:
Still waiting on final confirmation from Mihai, but I've removed firm related code based on the fact that firms do not do IAs.

Reply:
Yeah, I think we should be good with the existing code.  It will just create the new business identifier by calling the COLIN API.

![image](https://github.com/bcgov/lear/assets/6685223/79450b9f-c0ce-45f6-ad86-ec322262aea8)